Examples/Backend/DataAdapters/DataAdapter-SharePoint/Assets/Common/SharepointContentConverter.cs

```diff
@@ -324,6 +325,10 @@ private void SetThumbnails(AssetDataObject assetDataObject, DriveItem driveItem)
             {
                 var thumbnailSet = driveItem.Thumbnails?.FirstOrDefault();
 
+                assetDataObject.InternalMetadata.SmallThumbnailETag = GetETag(thumbnailSet?.Small?.Url);
```

Comment:
Is it possible in SharePoint to replace previews?

Reply:
I couldn't find a way to do this through their UI

Reply:
Then we do not need that code